daemon/lxcfs.c

```diff
@@ -128,6 +128,13 @@ lxcfs_daemon_start(const char *rt_path)
 		event_add_signal(sigchld);
 	}
 
+
```

Comment:
there could be a race, handle this in the sigchild handler above and set the is_active variable in error case to false, there and check it in exported functions such as the lxcfs_mount_proc_overlay())